qa/L0_perf_analyzer/test.sh

```diff
@@ -121,6 +123,12 @@ cp -r ../python_models/optional $DATADIR && \
   mv $DATADIR/optional/model.py $DATADIR/optional/1 && \
   sed -i 's/max_batch_size: 0/max_batch_size: 2/g' $DATADIR/optional/config.pbtxt
 
+# Copy decoupled model
+git clone --depth=1 https://github.com/triton-inference-server/python_backend
+mkdir -p $DATADIR/repeat_int32/1
```

Comment:
Use a different model in case the bug we're seeing with missing null response async callback is related to the model